src/covjson_pydantic/ndarray.py

```diff
@@ -9,17 +9,18 @@
 from .base_models import CovJsonBaseModel
 
 
-# TODO: Support for integers and strings
 class DataType(str, Enum):
     float = "float"
+    integer = "integer"
+    string = "string"
 
 
 class NdArray(CovJsonBaseModel, extra="allow"):
     type: Literal["NdArray"] = "NdArray"
     dataType: DataType = DataType.float  # noqa: N815
     axisNames: Optional[List[str]] = None  # noqa: N815
     shape: Optional[List[int]] = None
-    values: List[Optional[float]]
+    values: List[Optional[float | int | str]]
```

Comment:
Should we add a validator to check whether the type of `values` matches that of `dataType`?